pkg/verify/signed_entity.go

```diff
@@ -174,7 +174,7 @@ type TimestampVerificationResult struct {
 
 func NewVerificationResult() *VerificationResult {
 	return &VerificationResult{
-		Version: 20230823,
+		MediaType: "application/vnd.dev.sigstore.verificationresult+json;version=0.1",
```

Comment:
Nit but we should define the media type as a `const` so it's easier to work with.